news/2024/2024-12-05-osu-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "osu! World Cup 2024: Finals Recap"
+date: 2024-12-05 23:00:00 +0000
+---
+
+The dust has finally settled, and only 3 teams are left standing. Join us to refresh your memory, and prepare yourselves for an unparalleled showcase of skill this weekend.
+
+![](/wiki/Tournaments/OWC/2024/img/owc2024-banner.jpg)
+
+Finals week. The elite of the elite. Titans of tapping. Admirals of aim. Oligarchs of osu. Six teams entered, all with battles won in the rearview but with their sternest tests yet to come — three would come out the other side, heading for the grand finals, while three would taste bitter disappointment, eliminated so close to the finish line.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/-H-rOKG-2C4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Mappool recap
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+**[Download the pool here! (215 MB)](https://osu.ppy.sh/beatmaps/packs/P268)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2310246440&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![](/wiki/shared/news/2024-12-05-osu-world-cup-2024-finals-recap/owc-2024-f-mappool.jpg)
+
+The Finals stage pool was filled with bangers, some examples being the Hard Rock pick [roots](https://osu.ppy.sh/beatmapsets/2139515#osu/4502914) by [seros](https://osu.ppy.sh/users/10562853) acting as an aim pick with a sprinkle of alt, and [Aynane](https://osu.ppy.sh/beatmapsets/1842684#osu/3784901) (by one of our newest Featured Artists, [Zmey Gorynich](https://osu.ppy.sh/beatmaps/artists/450)) mapped by [Mazzerin](https://osu.ppy.sh/users/2942381), being the pool's main stream map. As is customary by now, the pool has its fair shared of old-school maps, one of them being [Blue Night](https://osu.ppy.sh/beatmapsets/31333#osu/116006), a classic 2011 map by [DoKiTo](https://osu.ppy.sh/users/537084), which when infused with Double Time becomes a challenge to keep up with! Looking at some more modern maps, [Lagtrain](https://osu.ppy.sh/beatmapsets/1313664#osu/2722655) also takes up a Double Time slot, although despite the song's popularity it wasn't picked at all this week.
+
+---
+
```

Comment:
i just noticed we've manage to alternate between having these and not having these. imo the banner acts fine as separator

```suggestion
```